pkg/analyzer/lib/src/generated/error_verifier.dart

```diff
@@ -1513,8 +1516,10 @@ class ErrorVerifier extends RecursiveAstVisitor<Object> {
               }
               if (!result.equalValues(_typeProvider, overriddenResult)) {
                 _errorReporter.reportErrorForNode(
-                    StaticWarningCode.INVALID_OVERRIDE_DIFFERENT_DEFAULT_VALUES_NAMED,
-                    formalParameters[i], [
+                    StaticWarningCode
```

Comment:
Yeah, there are some still remaining issues around collections that this is tickling. I wouldn't worry about it for now.